WORKSPACE

```diff
@@ -266,13 +264,9 @@ yarn_install(
 yarn_install(
     name = "npm_eslint_plugin_crdb",
     args = [
-        "--offline",
+        "--pure-lockfile",
```

Comment:
Can you additionally update the `Makefile` to use `--pure-lockfile` instead of `--offline`?

Reply:
Oh sure!  I forgot we're continuing to maintain that :)

Reply:
Done! Rebased on master and pushed again :)